get-started/part3.md

```diff
@@ -146,10 +146,41 @@ Now let's run it. You have to give your app a name. Here, it is set to
 docker stack deploy -c docker-compose.yml getstartedlab
 ```
 
-See a list of the 5 containers you just launched:
+Our single service stack is running 5 container instances of our deployed image
+on one host. Let's investigate.
+
+Get the service ID for the one service in our application:
+
+```shell
+docker service ls
+```
+
+Swarm for Docker runs tasks that spawn containers. Tasks have state and their
```

Comment:
It is my opinion that we could make this line clearer with something like
"Swarm for Docker runs tasks, which have state and their own IDs. In the case of Docker, each task is a container."

cc @mstanleyjones